pkg/cmd/server/origin/master.go

```diff
@@ -889,6 +897,13 @@ func (c *MasterConfig) RunDeploymentImageChangeTriggerController() {
 	controller.Run()
 }
 
+// SDN controller runs openshift-sdn if the said network plugin is provided
+func (c *MasterConfig) RunSDNController() {
+	if c.Options.NetworkConfig.NetworkPluginName == osdn.NetworkPluginName() {
```

Comment:
Yes, that flag defaults such that sdn is run, but can be disabled or pointed to another solution.
e.g. for the all-in-one case, its already disabled with the updated commit.